### PR TITLE
[TwigBridge] Update the undefined callable list

### PR DESCRIPTION
**Pull request by smnandre**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| License       | MIT

(minor/DX)

Add missing filter/functions (defined in Symfony\Bridge\Twig\Extension) in the UndefinedCallableHandler

Filter:
* serialize ([SerializerExtension](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Bridge/Twig/Extension/SerializerExtension.php))

Function:
* impersonation_path ([SecurityExtension](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Bridge/Twig/Extension/SecurityExtension.php))
* impersonation_url ([SecurityExtension](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Bridge/Twig/Extension/SecurityExtension.php))
* impersonate_exit_path ([SecurityExtension](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Bridge/Twig/Extension/SecurityExtension.php))
* impersonate_exit_url ([SecurityExtension](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Bridge/Twig/Extension/SecurityExtension.php))
* t ([TranslationExtension](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Bridge/Twig/Extension/TranslationExtension.php))

### CODE REVIEW

File: src/Symfony/Bridge/Twig/UndefinedCallableHandler.php [hunk 1]
```diff
@@ -59,6 +60,11 @@ class UndefinedCallableHandler
         'logout_url' => 'security-http',
         'logout_path' => 'security-http',
         'is_granted' => 'security-core',
+        'impersonation_path' => 'security-http',
+        'impersonation_url' => 'security-http',
+        'impersonate_exit_path' => 'security-http',
```
Comment (fabpot) [inline src/Symfony/Bridge/Twig/UndefinedCallableHandler.php]:
```suggestion
        'impersonation_exit_url' => 'security-http',
```

File: src/Symfony/Bridge/Twig/UndefinedCallableHandler.php [hunk 2]
```diff
@@ -59,6 +60,11 @@ class UndefinedCallableHandler
...
+        'impersonate_exit_url' => 'security-http',
```
Comment (fabpot) [inline src/Symfony/Bridge/Twig/UndefinedCallableHandler.php]:
```suggestion
        'impersonation_exit_url' => 'security-http',
```

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Bridge/Twig/UndefinedCallableHandler.php (hunk 1 above)
Comment (smnandre) [reply src/Symfony/Bridge/Twig/UndefinedCallableHandler.php]: Sorry for this

**Review by xabbuh — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @smnandre.

**Pull request merged**